packages/botonic-cli/tests/deploy.test.ts

```diff
@@ -0,0 +1,58 @@
+import { Config } from '@oclif/config'
+import { assert } from 'console'
+import { mkdtempSync, readdirSync } from 'fs'
+import { removeSync } from 'fs-extra'
+import { join } from 'path'
+import { chdir, cwd } from 'process'
+import rimraf from 'rimraf'
+
+import { BotonicAPIService } from '../src/botonic-api-service'
+import { EXAMPLES } from '../src/botonic-examples'
+import { default as DeployCommand } from '../src/commands/deploy'
+import { default as NewCommand } from '../src/commands/new'
+
+const botonicApiService = new BotonicAPIService()
+const newCommand = new NewCommand(process.argv, new Config({ root: '' }))
+const deployCommand = new DeployCommand(process.argv, new Config({ root: '' }))
+
+const BLANK_EXAMPLE = EXAMPLES[3]
+assert(BLANK_EXAMPLE.name === 'blank')
+
+describe('TEST: Deploy pipeline', () => {
+  test('Deploy', async () => {
```

Comment:
we should use a more descriptive name for the test: it should summarize test's intention, like in the other test cases added

Reply:
changed 👍